components/31-molecules/opening-hours/_opening-hours.scss

```diff
@@ -207,11 +207,11 @@ $week-breakpoint: '(min-width: 750px)';
       &--details {
         display: none;
         position: absolute;
-        width: calc(100% - #{$month-day-size-mobile / 2});
+        width: calc(100% - #{$month-day-size-mobile *.5});
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.